servlet/spring-boot/java/oauth2/azure-active-directory-b2c/README.adoc

```diff
@@ -0,0 +1,74 @@
+= Azure Active Directory B2C with Spring Security Sample
+
+In the current folder, we have the these samples to introduce how to use the Azure Active Directory B2C functions through Spring Security.
```

Comment:
By reading this tutorial, you will learn how to use Azure Active Directory B2C by Spring Security.